airflow/contrib/operators/ssh_execute_operator.py

```diff
@@ -142,7 +142,7 @@ def execute(self, context):
             logging.info("Output:")
             line = ''
             for line in iter(sp.stdout.readline, b''):
-                line = line.decode().strip()
+                line = line.decode('utf_8').strip()
```

Comment:
I think 'utf-8' is already the default encoding for `line.decode` -- is anything gained by making it explicit?